src/scss/components/button/mixins/_hxButton.scss

```diff
@@ -155,23 +155,23 @@
 }
 
 @mixin hxButton-active--secondary {
-  background-color: $cyan-700;
+  background-color: $cyan-900;
   border-color: transparent;
   color: $gray-0;
 }
 
 @mixin hxButton-disabled--secondary {
   background-color: transparent;
-  border-color: $gray-400;
-  color: $gray-400;
+  border: 1px solid $gray-500;
+  color: $gray-500;
 }
 
 @mixin hxButton-focus--secondary {
   box-shadow: $focus-glow;
 }
 
 @mixin hxButton-hover--secondary {
-  background-color: $cyan-500;
+  background-color: $cyan-700;
```

Comment:
Please update to the new `v2.0` color variables:

https://github.com/HelixDesignSystem/helix-ui/blob/eb547a4294af4f768294ffef085cfa58136718c1/src/scss/_vars.scss#L1-L118

Reply:
Changes have been done, Please review